solidity-v1/dashboard/src/components/Undelegations.jsx

```diff
@@ -91,6 +91,7 @@ const Undelegations = ({ undelegations, title }) => {
                     undelegation.managedGrantContractInstance
                   }
                   operatorAddress={undelegation.operatorAddress}
+                  amount={undelegation.amount}
```

Comment:
Can we also change button text of the recover button here? I think we decided to change it to 'Claim' (and it is like this also in Figma)

`btnText={"claim"}`

Reply:
b2bf118